package.json

```diff
@@ -18,6 +18,7 @@
     "classnames": "2.3.2",
     "eslint": "8.49.0",
     "eslint-config-next": "13.4.19",
+    "moment": "2.29.4",
```

Comment:
moment.js is not maintained anymore, day.js can be used

Ref: https://github.com/moment/moment#project-status